internal/packages/duty/yoda/collector/collector.go

```diff
@@ -21,12 +22,47 @@ var (
 // NOTE: this is for solo mode
 var packageMonikers []string
 
+// Due to the nature of how request misses are tracked and counted,
+// the list of all misses collected in the previous iteration is kept in memory
+var yodaRequestMisses = make([]types.ValidatorStatus, 0)
+
 const (
 	Subsystem      = "yoda"
-	SubsystemSleep = 60 * time.Second
+	SubsystemSleep = 10 * time.Second
```

Comment:
I needed to increase the scan interval as otherwise the requests can't be tracked due to how the request API response schema looks like. The schema does not include the delay or the block height a yoda oracle responded to the request, but will just add another entry in the `response:` array of the returned JSON. This means a request must be tracked and polled until it is expired to capture the amount of blocks an oracle needs to answer the request. If the polling interval is too long, too many blocks will already have passed and the captured result will have a bigger error margin. 
10s seems to be ok-ish at this equals to ~4blocks being produced in the meanwhile and is a small enough amount to still produce an usable metric, while still being able to use a public API endpoint. 

Alternatively the whole request tracking could be moved to a different `yoda_request` package, which could be turned off and on in the config, similar like you did with other "special" tracking options. 
In that case the original yoda status tracker could keep its interval while the new `yoda_request` package uses a different interval